lib/Core/Search/Legacy/Query/Common/CriterionHandler/ObjectStateIdentifier.php

```diff
@@ -35,6 +35,11 @@ public function accept(Criterion $criterion)
         return $criterion instanceof ObjectStateIdentifierCriterion;
     }
 
+    /**
+     * {@inheritdoc}
```

Comment:
We still do this? `@inheritdoc` is implicit. Even PHPStan doesn't need it.

Reply:
AFAIK it's implicit only if there are no additional annotations, like in this case.

Reply:
I suppose the deciding factor would be on how IDE behaves. I'll leave it to you :)